src/Sylius/Bundle/ApiBundle/Resources/config/services/query_handlers.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Sylius Sp. z o.o.
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd"
+>
+    <services>
+        <service id="Sylius\Bundle\ApiBundle\QueryHandler\GetCustomerStatisticsHandler">
+            <argument type="service" id="sylius.repository.customer" />
+            <argument type="service" id="sylius.customer_statistics_provider" />
+            <tag name="messenger.message_handler" bus="sylius.command_bus" />
+            <tag name="messenger.message_handler" bus="sylius_default.bus" />
```

Comment:
Is second tag needed?

Reply:
In that particular case no, but in case of command buses – yes, until `2.0`.